client/src/Components/FirstSection.css

```diff
@@ -0,0 +1,30 @@
+#marginRow {
+    margin-top: 5vh;
+    margin-bottom: 5vh;
+    font-family: roboto;
+}
+
+.img0Div{
```

Comment:
Can this be named something more descriptive so that it's easier to understand what it's meant to be used for?